c++/src/kj/memory.h

```diff
@@ -224,6 +224,8 @@ using PtrCounter = AtomicPtrCounter;
 template <typename T, typename StaticDisposer = decltype(nullptr)>
 class Own;
 
+void throwWrongDisposerError();
```

Comment:
Nitpick: Probably should be in `_` namespace, where we put everything that's not intended to be a public API...

```
namespace _ {  // (private)
void throwWrongDisposerError();
}  // namespace _ (private)
```